modules/nf-core/transdecoder/predict/main.nf

```diff
@@ -5,7 +5,7 @@ process TRANSDECODER_PREDICT {
     conda "${moduleDir}/environment.yml"
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
         'https://depot.galaxyproject.org/singularity/transdecoder:5.5.0--pl5262hdfd78af_4':
```

Comment:
Your Singularity image is still at version 5.5.0, causing the tests to fail.

Reply:
Aah, thanks!